### PR TITLE
Auto update copyright for auto exclusion

### PR DESCRIPTION
**Pull request by renfeiw**
Signed-off-by: renfeiw <renfeiw@ca.ibm.com>

### CODE REVIEW

File: scripts/testBot/playlistModifier.py [hunk 1]
```diff
@@ -103,15 +105,30 @@ def addDisabled(files, args):
...
         root = etree.parse(file)
+        updateCopyright(root, args)
         updateFile(file, root)
 
+def updateCopyright(root, args):
+    if (args["copyright"] == "true"):
+        cr = root.xpath(f"/comment()")
+        if cr:
+            crtext = cr[0].text
+            find = fr"(.*)(Copyright \(c\) \d\d\d\d, )(\d\d\d\d)(.*)"
```
Comment (llxia) [inline scripts/testBot/playlistModifier.py:124]: Could we do `(.*)(Copyright \(c\) \d{4}, )(\d{4}) (.*)`?

Comment (renfeiw) [reply scripts/testBot/playlistModifier.py]: updated, thanks!

**Comment by renfeiw** [comment]
reviewers: @smlambert @llxia

**Review by llxia — APPROVED** [review]
LGTM

**Review by smlambert — APPROVED** [review]
thanks @renfeiw !

**Comment by smlambert** [comment]
I think that the write access for @llxia who is one of the 2 reviewers is not quite trickled out yet, expect that the committers of the AQAvit project to have these permissions soon.

**Pull request merged**